packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -421,7 +421,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         // Add GC details to the summary.
         const gcDetails: IGarbageCollectionSummaryDetails = {
             usedRoutes: this.summarizerNode.usedRoutes,
-            gcData: summarizeResult.gcData,
```

Comment:
It might be worth putting comment somehwere around this code on differences between summarizeResult.gcData & this.summarizerNode.gcData, why they are different, and what are we going to do about them / any pending issues in this space

Reply:
I created a work item to remove gcData from summarize: https://github.com/microsoft/FluidFramework/issues/6223. I will add it to the comment here.